kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/PaymentRequestDocument.java

```diff
@@ -12,54 +10,69 @@
 import org.kuali.kfs.sys.context.SpringContext;
 import org.kuali.kfs.vnd.businessobject.VendorHeader;
-import org.kuali.rice.kew.framework.postprocessor.DocumentRouteStatusChange;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
 import org.kuali.rice.kew.api.KewApiConstants;
-import org.kuali.rice.krad.bo.Note;
+import org.kuali.rice.kew.api.WorkflowDocument;
+import org.kuali.rice.kew.framework.postprocessor.DocumentRouteStatusChange;
+import org.kuali.rice.kns.service.DocumentHelperService;
 import org.kuali.rice.krad.document.DocumentAuthorizer;
 import org.kuali.rice.krad.service.BusinessObjectService;
-import org.kuali.rice.kns.service.DocumentHelperService;
-import org.kuali.rice.krad.service.DocumentService;
-import org.kuali.rice.krad.service.NoteService;
 import org.kuali.rice.krad.util.GlobalVariables;
-import org.kuali.rice.core.api.util.type.KualiDecimal; 
 import org.kuali.rice.krad.util.ObjectUtils;
-import org.kuali.rice.kew.api.WorkflowDocument;
 
 import edu.arizona.kfs.fp.businessobject.PaymentMethod;
 import edu.arizona.kfs.fp.service.PaymentMethodGeneralLedgerPendingEntryService;
+import edu.arizona.kfs.module.purap.businessobject.PaymentRequestIncomeType;
 import edu.arizona.kfs.module.purap.service.PurapUseTaxEntryArchiveService;
+import edu.arizona.kfs.tax.document.IncomeTypeContainer;
+import edu.arizona.kfs.tax.document.IncomeTypeHandler;
 import edu.arizona.kfs.vnd.businessobject.VendorDetailExtension;
 
 
-public class PaymentRequestDocument extends org.kuali.kfs.module.purap.document.PaymentRequestDocument {
+public class PaymentRequestDocument extends org.kuali.kfs.module.purap.document.PaymentRequestDocument implements IncomeTypeContainer<PaymentRequestIncomeType, Integer> {
 
     private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(PaymentRequestDocument.class);
 
     public static final String DOCUMENT_TYPE_NON_CHECK = "PRNC";
     public static final String BANK = "bank";
 
+    private transient IncomeTypeHandler<PaymentRequestIncomeType, Integer> incomeTypeHandler;
+    private List<PaymentRequestIncomeType> incomeTypes;
+    private String paymentPaidYear;
+    private boolean payment1099Indicator;
 
     // default this value to "A" to preserve baseline behavior
     protected String paymentMethodCode = "A"; // check
     protected transient PaymentMethod paymentMethod;
     protected static PaymentMethodGeneralLedgerPendingEntryService paymentMethodGeneralLedgerPendingEntryService;
     
-    
+    public String getPaymentPaidYear() {
+        return paymentPaidYear;
+    }
+
+    public void setPaymentPaidYear(String paymentPaidYear) {
+        this.paymentPaidYear = paymentPaidYear;
+    }
+
+    public boolean isPayment1099Indicator() {
+        return payment1099Indicator;
+    }
+
+    public void setPayment1099Indicator(boolean payment1099Indicator) {
+        this.payment1099Indicator = payment1099Indicator;
+    }
+
```

Comment:
Disregard the last comment. There must have been other issues that were causing this to work incorrectly. The payment1099IndicatorForSearching bean has been added to the xml and the getter created in the Document.java file.